### PR TITLE
some changes in inline pmpermit + afk

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/plugins/utils/afk.py [hunk 1]
```diff
@@ -64,6 +72,10 @@ async def handle_afk_incomming(message: Message) -> None:
     """ handle incomming messages when you afk """
     user_id = message.from_user.id
     chat = message.chat
+    can_msg = False
+    if chat.type != 'private':
+        if await check_admin(chat.id, (await userge.get_me()).id):
+            can_msg = True
```
Comment (rking32) [inline userge/plugins/utils/afk.py:78]: sorry, i didn't get your point.

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py:78]: https://t.me/usergeot/638901

Comment (rking32) [reply userge/plugins/utils/afk.py:78]: i don't think this is the best solution for that

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py:78]: 😶😞 ohhk

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py:78]: just simple idea to avoid ban, bot will not reply in that group 🤷🤷

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py:78]: or suggest other way then 🤔🤔

Comment (rking32) [reply userge/plugins/utils/afk.py:78]: we can add blacklist or something like that to disable the userbot completely in some specific chats

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py:78]: with var or with command ?

Comment (rking32) [reply userge/plugins/utils/afk.py]: too much vars. we can implement it the way like notes, filters, ...

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py]: yeah

Comment (Krishna-Singhal) [reply userge/plugins/utils/afk.py]: Now merge if all Done.

File: userge/plugins/utils/pmpermit.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 import asyncio
 from typing import Dict
 from pyrogram.types import InlineKeyboardMarkup, InlineKeyboardButton, CallbackQuery
+from pyrogram.errors import BOT_INLINE_DISABLED
```
Comment (rking32) [inline userge/plugins/utils/pmpermit.py:14]: ImportError: cannot import name 'BOT_INLINE_DISABLED' from 'pyrogram.errors'

**Pull request merged**